packages/atomic/src/themes/coveo.css

```diff
@@ -5,6 +5,7 @@
   --atomic-primary-dark: #1a50ad;
   --atomic-on-primary: #ffffff;
   --atomic-ring-primary: rgba(19, 114, 236, 0.5);
+  --atomic-ring-primary-light: rgba(57, 159, 254, 0.3);
```

Comment:
This extra variable does not seem worth it, I'd prefer reusing `--atomic-ring-primary` & keep the number of theme variables to a minimum. Users will have more control if they chose

e.g.

```
atomic-popover {
  --atomic-ring-primary: rgba(57, 159, 254, 0.3)
}
```

Reply:
They'd have to add it on their own on their custom css file though?